code/game/machinery/computer/aifixer.dm

```diff
@@ -65,6 +65,7 @@
 	occupier.adjustOxyLoss(-5, 0, FALSE)
 	occupier.adjustFireLoss(-5, 0, FALSE)
 	occupier.adjustBruteLoss(-5, 0)
+	occupier.updatehealth()
```

Comment:
```suggestion
	occupier.adjustOxyLoss(-5, FALSE)
	occupier.adjustFireLoss(-5, FALSE)
	occupier.adjustBruteLoss(-5, FALSE)
	occupier.updateHealth()
```
the second arg of updateHealth is what allow dictates it to update (it's what i intended to do here, but I forgot to remove those damn zeros)

Reply:
I've cleaned up the proc call as suggested. A cosmetic change, though.

Probably the reason that the updating_health arg was set to FALSE is because it didn't want updatehealth() to be called 3 times for what is essentially a single heal operation.

Purely academic though considering the comment about #52391 in the review below.